torchgeo/transforms/temporal.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""TorchGeo temporal transforms."""
+
+from typing import Any
+
+from einops import rearrange
+from kornia.augmentation._3d.geometric.base import GeometricAugmentationBase3D
+from torch import Tensor
+
+
+class Rearrange(GeometricAugmentationBase3D):
+    """Rearrange tensor between [B, T, C, H, W] and [B, T*C, H, W]."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize a Rearrange instance.
+
+        Args:
+            *args: Positional arguments for einops.rearrange
+            **kwargs: Keyword arguments for einops.rearrange
```

Comment:
```suggestion
            *args: Positional arguments for :func:`einops.rearrange`.
            **kwargs: Keyword arguments for :func:`einops.rearrange`.
```
This should link to the einops docs directly. You will also need to add https://einops.rocks/ to `intersphinx_mapping` in `docs/conf.py`.